CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ All notable changes to this project will be documented in this file.
 
 - Add `context.redirect(path)` method to be used from inside route actions (PLANNED)
 
+### [v1.3.0] - [unreleased]
+
+- Handle `null` same way as `undefined`
+- Return `null` instead of `undefined` to signal no match. (BREAKING CHANGE)
```

Comment:
From semver:

> Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backwards compatible functionality is introduced to the public API. It MUST be incremented if any public API functionality is marked as deprecated. It MAY be incremented if substantial new functionality or improvements are introduced within the private code. It MAY include patch level changes. Patch version MUST be reset to 0 when minor version is incremented.

I think this is the case